internal/plugincommon/consensus/consensus.go

```diff
@@ -83,6 +83,37 @@ func Median[T any](vals []T, less func(T, T) bool) T {
 	return valsCopy[len(valsCopy)/2]
 }
 
+// GetMaxSqNrWithConsensus returns the greatest sequence number for each chain that meets the min obs threshold
+// We do this by sorting the array in ascending order, then returning the value of item[minObs]
+func GetMaxSqNrWithConsensus[K comparable](
```

Comment:
This isn't seq nr specific right? Can work for any `V comparable`, since you just need to `sort.Slice` something?